executor/set_test.go

```diff
@@ -896,9 +896,9 @@ func (s *testSuite5) TestValidateSetVar(c *C) {
 	result = tk.MustQuery("select @@tmp_table_size;")
 	result.Check(testkit.Rows("167772161"))
 
-	tk.MustExec("set @@tmp_table_size=18446744073709551615")
+	tk.MustExec("set @@tmp_table_size=9223372036854775807")
```

Comment:
This change is because MySQL set the value to Uint64, while in TiDB it's set to Int64